release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2477 @@
+====
+
+[[ocp-36-installation-of-etcd-docker-daemon-and-ansible-installer-as-system-containers]]
+==== Installation of etcd, Docker Daemon, and Ansible Installer as System Containers (Technology Preview)
+
+RHEL System Containers offer more control over the life cycle of the services
+that do not run inside {product-title} or Kubernetes. Additional system
+containers will be offered over time.
+
+System Containers leverage the OSTree on RHEL or Atomic Host. They are
+controlled by the kernel init system and therefore can be leveraged earlier in
+the boot sequence. This feature is enabled in the installer configuration.
+
+[IMPORTANT]
+====
+This feature (currently in xref:ocp-36-technology-preview[Technology Preview)] is
+currently not for production workloads and must be used in a non-production
+sandbox.
+====
+
+For more information, see
+xref:../install_config/install/advanced_install.adoc#advanced-install-configuring-system-containers[Configuring
+System Containers].
+
+[[ocp-36-running-openshift-installer-as-a-system-container]]
+==== Running OpenShift Installer as a System Container (Technology Preview)
+
+[IMPORTANT]
+====
+This feature (currently in xref:ocp-36-technology-preview[Technology Preview)] is
+currently not for production workloads and must be used in a non-production
+sandbox.
+====
+
+To run the {product-title} installer as a system container:
+
+----
+$ atomic install --system --set INVENTORY_FILE=$(pwd)/inventory registry:port/openshift3/ose-ansible:v3.6
+
+$ systemctl start ose-ansible-v3.6
+----
+
+[[ocp-36-etcd3-model-for-new-installations]]
+==== etcd3 Data Model for New Installations
+
+Starting with new installations of {product-title} 3.6, the etcd3 v3 data model
+is the default. By moving to the etcd3 v3 data model, there is now:
+
+- Larger memory space to enable larger cluster sizes.
+- Increased stability in adding and removing nodes in general life cycle actions.
+- A significant performance boost.
+
+A migration playbook will be provided in the near future allowing
+upgraded environments to migrate to the v3 data model.
+
+[[ocp-36-cluster-wide-control-of-ca]]
+==== Cluster-wide Control of CA
+
+You now have the ability to change the certificate expiration date en mass
+across the cluster for the various framework components that use TLS.
+
+We offer new cluster variables per framework area so that you can use different
+time-frames for different framework components. Once set, issue the new
+`redeploy-openshift-ca` playbook.
+
+.New Cluster Variables
+----
+# CA, node and master certificate expiry
+openshift_ca_cert_expire_days=1825
+openshift_node_cert_expire_days=730
+openshift_master_cert_expire_days=730
+
+# Registry certificate expiry
+openshift_hosted_registry_cert_expire_days=730
+
+# Etcd CA, peer, server and client certificate expiry
+etcd_ca_default_days=1825
+
+$ ansible-playbook -i inventory openshift-ansible/playbook/byo/openshift-cluster/redeploy-openshift-ca.yml
```

Comment:
As comments before, this playbook - redeploy-openshift-ca.yml is only one of the 3 main playbooks used for redeploying certificates. Better to get rid of this line.